quickwit-cli/src/index.rs

```diff
@@ -53,6 +54,15 @@ pub struct DescribeIndexArgs {
     pub index_id: String,
 }
 
+impl DescribeIndexArgs {
+    pub fn new(metastore_uri: String, index_id: String) -> anyhow::Result<Self> {
```

Comment:
I'm on @fulmicoton's side regarding `new` constructors: I avoid them when the attributes are `pub`, and the object's construction is straightforward.

Reply:
agreed, don't know why there is still a new there, I try to avoid them as much as possible.

Reply:
mm, coming from the rebase :)